horace_core/sqw/coord_transform/@aProjectionBase/calc_pbins.m

```diff
@@ -76,20 +76,20 @@
 % ------------------------------------------------------------------------
 n=length(pbin);
 if n<3 || n>4
-    error('aPROJECTION:invalid_arguments',...
+    error('HORACE:aProjectionBase:invalid_arguments',...
         'Must provide binning descriptor for all three momentum axes and (optionally) the energy axis');
 end
 
 if ~(isempty(pbin{1})||(isa_size(pbin{1},'row','double') && length(pbin{1})>=1 && length(pbin{1})<=4)) || ...
         ~(isempty(pbin{2})||(isa_size(pbin{2},'row','double') && length(pbin{2})>=1 && length(pbin{2})<=4)) || ...
         ~(isempty(pbin{3})||(isa_size(pbin{3},'row','double') && length(pbin{3})>=1 && length(pbin{3})<=4))
```

Comment:
Would prefer this check to be a sensibly named function, then checked here through `all(cellfun(...))` particularly because it's also used repeatedly below. But not for this PR

Reply:
This is very old code which probably should go. I wanted to deleted it but this is used by cut_sym only so the time for changes have not come.